common/src/main/java/com/kin/ecosystem/common/ObservableData.java

```diff
@@ -117,4 +117,8 @@ private void onChanged() {
             observer.onChanged(value.get());
         }
     }
+
+    public void removeAllObservers() {
```

Comment:
this is dangerous.  one observer may cancel to all other

Reply:
what do you suggest?

Reply:
why do you want to remove observers? you can disable and enable the observer maybe?

Reply:
well, as i see it now, we allowing the developer to add observers although the user is not logged in so we should not remove them. i'll change that.

Reply:
I'm not sure we need to remove observer - if we log out this observer should not omit any value